Description: Fix a couple of compiler warnings.
 - undefine the asprintf macro if it is already defined, e.g. in glibc
Author: Peter Pentchev <roam@ringlet.net>
Forwarded: not yet
Last-Update: 2013-01-25

--- a/lib/util/asprintf.h
+++ b/lib/util/asprintf.h
@@ -2,6 +2,9 @@
 #define _ASPRINTF_H_
 
 /* Avoid namespace collisions with BSD/GNU asprintf. */
+#ifdef asprintf
+#undef asprintf
+#endif
 #define asprintf compat_asprintf
 
 /**
